srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -295,7 +298,7 @@ def create_ps_ff_srf(
     shypo = parameter_dictionary.pop("shypo")
     dhypo = parameter_dictionary.pop("dhypo")
     genslip_version = str(parameter_dictionary.pop("genslip_version"))
-    seed = parameter_dictionary.pop("seed", get_seed())
+    seed = parameter_dictionary.pop("srfgen_seed")
```

Comment:
This no longer exists in the dictionary?

Reply:
What do you mean? The old behaviour was to take from the dict if there, otherwise call get_seed, the new behaviour is to take from the dict and raise an exception if not there.

Reply:
It's popped a few hundred lines above in the PR without saving it.

Reply:
I might have thought that those functions were the same but doesn't look like it.

Reply:
Yeah, that'll be the type 1 func that doesn't need it, but needs to pop it anyway. The type 2 func here needs it though

Reply:
We can actually use seeds in type 1 generation if we perturbate rise time using Rob Graves perturbations in slip2srf